OloEngine/src/OloEngine/Networking/Replication/NetworkInterestManager.cpp

```
@@ -0,0 +1,208 @@
+#include "OloEnginePCH.h"
+#include "NetworkInterestManager.h"
+#include "OloEngine/Scene/Scene.h"
+#include "OloEngine/Scene/Entity.h"
+#include "OloEngine/Scene/Components.h"
+#include "OloEngine/Debug/Profiler.h"
+
+#include <glm/gtx/norm.hpp>
+
+namespace OloEngine
+{
+    void NetworkInterestManager::SetClientPosition(u32 clientID, const glm::vec3& position)
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'SetClientPosition' of similar type are easily swapped by mistake

(bugprone-easily-swappable-parameters)